openshift/console
# ODC-6127: Fix missing build status translation in topology sidebar

**Fixes**: 
https://issues.redhat.com/browse/ODC-6127

**Analysis / Root cause**: 
Generated strings in translation doesn't match expectation from `Trans` component.

**Solution Description**: 
Remove the {link} and replace it with the component so that generated translation and code matches again.

Searched in our translation files for other cases (regex: `[^{]\{[a-z]`) and doesn't find any other case.

**Screen shots / Gifs for design review**: 
Before:
![broken-translation-before](https://user-images.githubusercontent.com/139310/126526649-99ac40b6-2180-43fb-9986-cbd36df6013d.png)

After:
![fixed-translation](https://user-images.githubusercontent.com/139310/126526659-8ec21785-361c-4328-be6e-2a6ad7c755a3.png)

**Unit test coverage report**: 
Unchanged.

**Test setup:**
1. Create a Deployment with a BuildConfig (via import from git for example)
2. Open topology
3. Select the Deployment and check the sidebar.
4. The "Resources" tab contains a "Builds" section
5. Switch to another language to see if the info Build #x was complete is translated correctly.

**Browser conformance**:
